pkgs/games/spring/default.nix

```diff
@@ -68,41 +56,36 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake makeWrapper docbook_xsl docbook_xsl_ns asciidoc ];
   buildInputs = [
-    xz
     boost
+    curl
+    freetype
+    glew
     libdevil
-    zlib
-    p7zip
-    openal
+    libGL
+    libGLU
+    libunwind
     libvorbis
-    freetype
+    minizip
+    openal
+    p7zip
     SDL2
     xorg.libX11
     xorg.libXcursor
-    libGLU
-    libGL
-    glew
-    curl
-    systemd
-    libunwind
-    which
-    minizip
+    xz
+    zlib
   ]
-  ++ lib.optional withAI jdk
-  ++ lib.optional withAI python;
-
-  NIX_CFLAGS_COMPILE = "-fpermissive"; # GL header minor incompatibility
+  ++ lib.optionals withAI [ python3 jdk ];
 
   postInstall = ''
     wrapProgram "$out/bin/spring" \
-      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ stdenv.cc.cc systemd ]}"
```

Comment:
Searched in the repo and didn't find any mention of systemd usage except for an unrelated service that they use in their buildbot. Also the games looks like it works fine without it.